runner/README.md

```diff
@@ -1,6 +1,6 @@
-# Package server
+# Package runner
 
-`package server` provides the types needed to run application servers.
+`package runner` provides the types needed to run stuff.
```

Comment:
I would extend this description with more details, i.e. explaining why it's important to run stuff this way